var/spack/repos/builtin/packages/wgl/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import re
+
+from spack.package import *
+
+
+class Wgl(BundlePackage):
+    """External WGl and Windows OpenGL emulation representation in Spack"""
```

Comment:
If this isn't meant to be installed, then I recommend using the same technique as for `win-sdk` (raise an error): if `wgl` is not present on the system, then - with this package in its current state - `spack install wgl` wouldn't actually install the `wgl` libraries.

Given the dependency version locking you do here, I assume the question would then be how to associate the right `wgl` with the correct `win-sdk` implementation. That is a more general issue for how we handle externals and for now we can assume that the `packages.yaml` will contain a single instance of `wgl` and `win-sdk` (or at least, if that isn't true then that's a problem in Spack core that we need to address and not specifically an issue with this PR).

Reply:
I was following the existing pattern for opengl style packages (WGL is essentially EGL), which lead me to believe bundle packages also could not be installed. I agree, and will refactor to have WGL throw during the install method. I don't believe this requires changes beyond that, but please correct me if that's not the case.